mesh/mesh.cpp

```diff
@@ -11220,14 +11220,13 @@ int Mesh::FindPoints(DenseMatrix &point_mat, Array<int>& elem_ids,
 }
 
 
-GeometricFactors::GeometricFactors(const Mesh *mesh, const IntegrationRule &ir,
+GeometricFactors::GeometricFactors(const Mesh &mesh, const IntegrationRule &ir,
```

Comment:
I believe these interfaces are internal enough that changing them won't be a big problem.  @tzanio do you agree?

Reply:
We may need to check this. I am not sure if these are not used in Laghos and other applications.